Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -22,13 +22,21 @@ def install
     (buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve").install Dir["{*,.git,.gitignore}"]
 
     ENV["GOPATH"] = "#{buildpath}/gopath"
-
-    cd buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve" do
+    build_root = buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve"
+    cd build_root do
       if build.head?
         git_hash = `git rev-parse --short HEAD --quiet`.chomp
-        git_hash = " HEAD-#{git_hash}"
+        git_hash = "HEAD-#{git_hash}"
+        ENV["CGO_LDFLAGS"] = "#{build_root}/vendor/build/lib9p/lib9p.a -L#{build_root}/vendor/lib9p"
+        ENV["CGO_CFLAGS"] = "-I#{build_root}/vendor/lib9p"
+        mkdir "vendor/build"
```

Comment:
Sorry for the slow reply; not around Homebrew a huge amount this past week. Let's shorten it a bit:
```ruby
(buildpath/"vendor/build/lib9p/backend").mkpath
(buildpath/"vendor/build/lib9p/transport").mkpath
(buildpath/"vendor/build/lib9p/sbuf").mkpath
```
That saves us two extra lines. If this is being addressed upstream as well it'd be nice to include a link to that PR/Issue so we know when it's safe to remove.